cfme/tests/automate/test_automate_manual.py

```diff
@@ -1025,3 +1025,115 @@ def test_automate_task_schedule():
         1713072
     """
     pass
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1693362])
+def test_redhat_domain_sync_after_upgrade():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        testSteps:
+            1. Dump database of appliance(5.10) to appliance(5.11) or Upgrade 5.10 appliance to 5.11
```

Comment:
Hey @john-dupuy 
- I agree; I should not mention appliance version (It may vary)
- Hence I update that sentence to this: ```Either dump database of appliance with version X to appliance with version Y  or upgrade the appliance```